custom/icds_reports/templates/icds_reports/icds_app/mobile/indie-map.directive.html

```diff
@@ -0,0 +1,5 @@
+<div class="indie-map-directive">
+  <div id="locPopup" class="locPopup"></div>
+  <datamap on-click="$ctrl.handleMapClick" map="$ctrl.map" plugins="$ctrl.mapPlugins" zoomable
```

Comment:
the `zoomable` is the only difference between these two files. unfortunately passing `zoomable=false` still causes it to be zoomable otherwise i'd just pass it through a controller property...